src/Elders.Cronus/Hosting/ProjectionsBootstrapper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Elders.Cronus.Pipeline.Config;
+using Elders.Cronus.Projections.Versioning;
+
+namespace Elders.Cronus
+{
+    public class ProjectionsBootstrapper
+    {
+        private readonly HandlerTypeContainer<IProjection> projectionTypes;
+        private readonly ProjectionHasher hasher;
+        private readonly IPublisher<ICommand> publisher;
+
+        public ProjectionsBootstrapper(HandlerTypeContainer<IProjection> projectionTypes, ProjectionHasher hasher, IPublisher<ICommand> publisher)
+        {
+            this.projectionTypes = projectionTypes;
+            this.hasher = hasher;
+            this.publisher = publisher;
+        }
+
+        public void Bootstrap()
+        {
+            foreach (var handler in projectionTypes.Items)
+            {
+                var id = new ProjectionVersionManagerId(handler.GetContractId());
+                var command = new RegisterProjection(id, hasher.CalculateHash(handler).ToString());
+                publisher.Publish(command);
```

Comment:
What happens if the publisher fails?

Reply:
The command is not published and there is a small chance something to not work